### PR TITLE
#3 This is my attempt at prettying up the modal

### LINKED ISSUE #3: Improve look of Delete Modal
Right now the Modal looks too dull, would be nice to add colors for the CTA buttons and more padding to the modal

### PR DESCRIPTION
**Pull request by dennisxu54**
closes #3

### CODE REVIEW

**Review by egodihc — CHANGES_REQUESTED** [review]
Almost there, try making the modal taller and add hover effects to the buttons (change the cursor or change button color to give visual feedback)

File: src/App.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const App = () => {
   const filteredPosts = filterPosts(items, searchQuery);
   const [show, setShow] = useState(false)
   const [deleteUser, setDeleteUser] = useState(0)
+  const [robotName, setRobotName] = useState("")
```
Comment (egodihc) [inline src/App.js]: This variable name should be more meaningful, something like `robotToDelete` or something, just `robotName` might be a bit too vague to people reading the code.
Although, it is actually possible to pass the robot name to the modal without using this state. Have a think about what else we can recycle :)

File: src/components/LoadList/LoadList.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const LoadList = ({ items, error, onOpen }) => {
         <h2>{item.username}</h2>
         <h3>{item.email}</h3>
         <button className="delete-button" onClick={() => {
-          onOpen(index) 
+          onOpen(index, item.name)
```
Comment (egodihc) [inline src/components/LoadList/LoadList.js]: We can also remove this extra argument if we're implementing the suggestion above :)

**Review by egodihc — CHANGES_REQUESTED** [review]
Overall, pretty good changes, just a few optimizations and we're good :)

**Review by egodihc — CHANGES_REQUESTED** [review]
The idea is there, but what would happen if two users happened to have the same name?
There is a better property we can pass to `deleteUser`, you've used it before

File: src/App.js [hunk 3]
```diff
@@ -45,23 +45,23 @@ const App = () => {
...
-    setDeleteUser(user)
-    setShow(true)
+    setDeleteUser(user);
+    setShow(true);
   }
 
   function deleteItem() {
-    setItems(items.filter((item) => item.name !== deleteUser));
-    setShow(false)
+    setItems(items.filter((item) => item !== deleteUser));
```
Comment (egodihc) [inline src/App.js]: When comparing objects, best to use `Object.is(object1, object2)`

File: src/App.js [hunk 4]
```diff
@@ -23,8 +23,8 @@ const App = () => {
   const [items, setItems] = useState([]);
   const [searchQuery, setSearchQuery] = useState("");
   const filteredPosts = filterPosts(items, searchQuery);
-  const [show, setShow] = useState(false)
-  const [deleteUser, setDeleteUser] = useState("")
+  const [show, setShow] = useState(false);
+  const [deleteUser, setDeleteUser] = useState(0);
```
Comment (egodihc) [inline src/App.js]: Type inconsistency - using `number` here but the variable is actually of type `user`
Will cover this when we go over TypeScript :)

Comment (dennisxu54) [reply src/App.js]: Do i just set it to empty or null then?

File: src/App.js (hunk 3 above)
Comment (egodihc) [reply src/App.js]: But ideally, you would just be using the `index` anyway to check which item to delete, not the entire object.

**Review by egodihc — CHANGES_REQUESTED** [review]
The idea was to use the `index` instead of passing the entire object, this way, if you wanted to pass the `name` to the modal, you could instead pass `items[deleteUser].name`

File: src/App.js (hunk 4 above)
Comment (egodihc) [reply src/App.js]: You can leave it as empty

Comment (egodihc) [reply src/App.js]: Once we go over TypeScript, you will learn the better convention for this

**Review by egodihc — APPROVED** [review]
LGTM! Loving the new variable names 
image

**Pull request merged**